backend/src/main/java/com/google/sps/notifiers/EmailNotifier.java

```diff
@@ -0,0 +1,49 @@
+// Class used to send users emails regarding their status in regards to their Ad-Lib Meeting.
+
+import com.sendgrid.*;
+import com.sendgrid.helpers.mail.Mail;
+import com.sendgrid.helpers.mail.objects.*;
+
+import java.io.IOException;
+
+
+
+public class EmailNotifier extends Notifier {
+
+    // Member Variables 
+    
+    public String toEmail ;       
+    public String fromEmail;
+
+    public EmailNotifier(String Name, String fromEmail, String toEmail)
```

Comment:
nit: Can this be lowercase "name" instead? Here and everywhere else?

Reply:
Done